ExtractorUtils/Cognite/DestinationUtils.cs

```diff
@@ -242,6 +240,23 @@ public static void ConfigureSslPolicy(CertificateConfig config)
             {
                 throw new CogniteUtilsException("Cannot configure Builder: Project is not configured");
             }
+
+            string? _tenant = config.IdpAuthentication?.Tenant.TrimToNull();
+            string? _tokenUrl = config.IdpAuthentication?.TokenUrl.TrimToNull();
```

Comment:
underscore prefixed variables are reserved for private members, use `tenant` and `tokenUrl` instead.